cmd/skywire-visor/commands/root.go

```diff
@@ -173,22 +170,20 @@ var rootCmd = &cobra.Command{
 			os.Exit(1)
 		}
 		//log for initial checks
-		log := initLogger(tag, syslogAddr)
-		_, hook := logstore.MakeStore(runtimeLogMaxEntries)
-		log.AddHook(hook)
+		mLog := initLogger(tag, syslogAddr)
+		log := mLog.PackageLogger("pre-run")
+
+		//_, hook := logstore.MakeStore(runtimeLogMaxEntries)
```

Comment:
Please remove commented code if you want to remove it.

Reply:
done